### DISCUSSION TITLE
中文字幕水印打不上打不全的问题

### DISCUSSION THREAD

**Discussion by Kaystar** [metatube-community/jellyfin-plugin-metatube#180] [Q&A]
前几天添加了六七十部某堂中文字幕的片子，刮削完之后发现打上中文字幕水印的只有不到30部

1.插件设置中已勾选Enable badges，Badge url: 是默认设置
2.片子命名正确:XXXX-yyy-C/XXXX-yyy-C.mp4
3.计划任务中已多次运行Organize Metadata，甚至重新刮削了多次，均未能解决

然后又查查了以前的片子，发现也有很多没打上字幕水印的，包括正确命名的外挂字幕的片子
麻烦有经验的大佬帮帮忙。

**Comment by JassonML** [comment]
看到有人跟我一样，我就放心了。。。两年了，我依然还在寻求解决之法
